crates/data_components/src/databricks/sql_warehouse.rs

```diff
@@ -89,6 +89,9 @@ pub enum Error {
 
     #[snafu(display("Failed to parse Databricks datatype: {reason}"))]
     ParseError { reason: String },
+
+    #[snafu(display("Query failure: {message}"))]
```

Comment:
```suggestion
    #[snafu(display("Failed to execute the query.\n{message}\nVerify the query is valid, or report a bug at: https://github.com/spiceai/spiceai/issues"))]
```

Please review the [error handling guide](https://github.com/spiceai/spiceai/blob/trunk/docs/dev/error_handling.md).